packages/node_modules/@webex/plugin-messages/test/integration/spec/messages.js

```diff
@@ -377,6 +377,79 @@ describe('plugin-messages', function () {
           });
       });
     });
+
+    describe('#threadedMessageTests()', () => {
```

Comment:
The `#function()` format is used to describe a specific method one would call. 

```suggestion
    describe('threaded message support', () => {
```